jvb/src/main/kotlin/org/jitsi/videobridge/EndpointConnectionStatusMonitor.kt

```diff
@@ -92,6 +92,11 @@ class EndpointConnectionStatusMonitor @JvmOverloads constructor(
                 return
             } else {
                 logger.cdebug { "${endpoint.id} not ready for activity checks yet" }
+                // It's possible this endpoint was previously on the inactive list but has
+                // been recreated, so try removing it from the inactive list just in case.
+                synchronized(inactiveEndpointIds) {
+                    inactiveEndpointIds -= endpoint.id
```

Comment:
Okay, I'm rethinking this now...if we removed it from the list we should notify the other endpoints (otherwise the update will not be triggered when it connects). Maybe we should keep it in the list until it either connects or times out?

Reply:
When it reconnects, we won't have sent any status update for it to the other endpoints, right?  And they assume 'active' by default?  If this is too sticky, maybe it's worth doing this fix separately.

Reply:
I'm thinking: 
1. endpoint connects
2. it loses connectivity, we send "inactive" to other endpoints
3. it signals to jicofo that ice failed, jicofo re-creates it
4. it connects again (as a new Endpoint instance, but already in the `inactiveEndpointIds` list

Reply:
Ah ok, I thought endpoints were active by default until the bridge signaled otherwise to the clients so merely removing it from the `inactiveEndpoints` list would be sufficient, but yes I see now we query `inactiveEndpoints` when an EP joins.  How about I take this commit off now so we can merge the first fix and we can think about how we want to handle this case?

Reply:
Sure.